docs/configure/triggers.md

```diff
@@ -204,8 +255,114 @@ The following attributes are available for triggers:
+| `notifications` |  object | **Required** | The notifications object: <ul><li>`type`: The type of the notification. Options: `webhook`, `email`</li><li>`value`: The URL to send the request to or the email address to notify.</li><li>`seconds_between_notifications`: The interval between notifications in seconds.</li></ul> |
+
+#### Conditions
+
+The `conditional` object for the `conditional_data_ingested` trigger includes the following options:
+
+<!-- prettier-ignore -->
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| `data_capture_method` | string | **Required** | The method of data capture to trigger on. <br> Example: `sensor:<name-of-component>:Readings`. |
+| `condition` | object | Optional | Any additional conditions for the method to fire the trigger. Leave out this object for the trigger to fire any time there is data synced. <br> Options: <ul><li>`evals`:<ul><li>`operator`: Logical operator for the condition. </li><li>`value`: Can be a number, string or another object like a key-value pair that specifies the shape of the readings. </li></ul></li></ul> |
+
+Options for `operator`:
+
+| Name  | Description              |
+| ----- | ------------------------ |
+| `lt`  | Less than                |
+| `gt`  | Greater than             |
+| `lte` | Less than or equal to    |
+| `gte` | Greater than or equal to |
+| `eq`  | Equals                   |
+| `neq` | Does not equal           |
+
+Examples:
+
+{{< tabs >}}
+{{% tab name="No nesting" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"condition": {
+  "evals": [
+    {
+      "operator": "lt",
+      "value": 20
+    }
+  ]
+}
+```
+
+This eval would trigger for the following sensor reading:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "readings": {
+    "key": 10
+  }
+}
+```
+
+{{% /tab %}}
+{{% tab name="1 level of nesting" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"condition": {
+  "evals": [
+    {
+      "operator": "lt",
+      "value": {
+        "Line-Neutral AC RMS Voltage": 130
+      }
+    }
+  ]
+}
+```
+
+This eval would trigger for the following sensor reading:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "readings": {
+    "Line-Neutral AC RMS Voltage": 100
+  }
+}
+```
+
+{{% /tab %}}
+{{% tab name="2 levels of nesting" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"condition": {
+  "evals": [
+    {
+      "operator": "lt",
+      "value": {
+        “level_1_of_object”: {
+          “level_2_of_object”: 50
```

Comment:
This uses smart quotes - you must've copied it from somewhere that's why the formatting is weird (and code would probably not run)
```suggestion
        "level_1_of_object": {
          "level_2_of_object": 50
```